src/coverlet.console/Program.cs

```diff
@@ -15,6 +15,42 @@ namespace Coverlet.Console
 {
     class Program
     {
+        /// <summary>
+        /// Exit Codes returned from Coverlet console process.
+        /// </summary>
+        enum CommandExitCodes
```

Comment:
This should be a `[Flags]` i.e. https://www.ideone.com/9OvttX https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netframework-4.8

Reply:
Cool, I forget about it. I can remove `TestFailedAndCoverageBelowThreshold` from enum with this.